packit_service/worker/build/babysit.py

```diff
@@ -76,6 +71,13 @@ def check_pending_testing_farm_runs() -> None:
             created,
         ) = Parser.parse_data_from_testing_farm(run, details)
 
+        logger.info(
+            f"Result for the testing farm pipeline {run.pipeline_id} is {result}."
+        )
+        if result == TestingFarmResult.running:
```

Comment:
To me it seems that this should also include "new" and "queued", as these are also _not completed_ states. Opinions?

Reply:
Good catch, using `result in (TestingFarmResult.new, TestingFarmResult.queued, TestingFarmResult.running)` makes more sense

Reply:
Updated. PTAL.